nikola/nikola.py

```diff
@@ -1984,6 +1984,20 @@ def create_hierarchy(cat_hierarchy, parent=None):
         # Next, flatten the hierarchy
         self.category_hierarchy = utils.flatten_tree_structure(root_list)
 
+    def sort_posts(self, posts, lang=None):
```

Comment:
There seems to be no way to end up with the unsorted list, so you can leave it as-is.